src/software/thunderscope/robot_diagnostics/robot_info.py

```diff
@@ -351,3 +368,11 @@ def __update_ui(self, robot_status: RobotStatus) -> None:
         )
 
         self.battery_progress_bar.setValue(power_status.battery_voltage)
+
+    def __calculate_average_round_trip_time(self, new_time: float) -> int:
+        self.previous_primitive_rtt_values.append(new_time)
+        average_rtt_time_milliseconds = 0
+        for value in self.previous_primitive_rtt_values:
+            average_rtt_time_milliseconds += value
+        return int(average_rtt_time_milliseconds/len(self.previous_primitive_rtt_values))
```

Comment:
nit: this doesn't really matter, but a neat little feature in Python is being able to do integer division using `//`. So you can have: `average_rtt_time_milliseconds // len(self.previous_primitive_rtt_values)`

Reply:
I gave this a shot and learned something interesting, apparently the floor operator actually produces a float when one of the operands is a float. https://www.learndatasci.com/solutions/python-double-slash-operator-floor-division/

Reply:
Interesting, good find!